packages/coverage-v8/src/index.ts

```diff
@@ -23,15 +24,19 @@ export default {
     })
   },
 
-  takeCoverage(): Promise<{ result: Profiler.ScriptCoverage[] }> {
+  takeCoverage({ moduleExecutionInfo }): Promise<{ result: Profiler.ScriptCoverage[] }> {
```

Comment:
a little type safety

```suggestion
  takeCoverage({ moduleExecutionInfo }): Promise<{ result: ScriptCoverageWithOffset[] }> {
```